10-std.tex

```diff
@@ -368,6 +368,98 @@ \section{Lambda}
 }
 \end{lstlisting} 
 
+\section{Template}
+\label{std-template}
+
+Haxe comes with a standard template system with an easy to use syntax which is interpreted by a lightweight class called \type{haxe.Template}.
+
+A template is a string or a file that is used to produce any kind of string output depending on the input. Here is a small template example:
+
+\haxe{assets/Template.hx}
+
+The console will trace \ic{My name is Mark, 30 years old}.
+
+\paragraph{Expressions}
+An expression can be put between the \ic{::}, the syntax allows the current possibilities:
+
+\begin{description}
+	\item[\ic{::name::}] the variable name
+	\item[\ic{::expr.field::}] field access
+	\item[\ic{::(expr)::}] the expression expr is evaluated
+	\item[\ic{::(e1 op e2)::}] the operation op is applied to e1 and e2
+	\item[\ic{::(135)::}] the integer 135. Float constants are not allowed
+\end{description}
+
+\paragraph{Conditions}
+It is possible to test conditions using \ic{::if flag1::}. Optionally, the condition may be followed by \ic{::elseif flag2::} or \ic{::else::}. Close the condition with \ic{::end::}.
+
+\begin{lstlisting} 
+::if isValid:: valid ::else:: invalid ::end::
+\end{lstlisting} 
+
+Operators can be used, but it doesn't deal with operator precedence. Therefore it is required to enclose each operation in parentheses \ic{()}. The current operators are allowed: \ic{+}, \ic{-}, \ic{*}, \ic{/}, \ic{>}, \ic{<},  \ic{>=}, \ic{<=}, \ic{==}, \ic{!=}, \ic{&&} and \ic{||}.
```

Comment:
no comma before "but", also "it" refers to what?
"Currently, the following operators are allowed:"